src/display/api.js

```diff
@@ -981,9 +987,10 @@ class PDFDocumentProxy {
  * Proxy to a PDFPage in the worker thread.
  */
 class PDFPageProxy {
-  constructor(pageIndex, pageInfo, transport, pdfBug = false) {
+  constructor(pageIndex, pageInfo, transport, ownerDocument, pdfBug = false) {
```

Comment:
Should we use the `ownerDocument = globalThis.document` fallback here too since the parameter is optional?

Reply:
it probably shouldn't be optional internally, and only optional on public api

Reply:
That seems alright. Thanks!